acceptance/openstack/blockstorage/v3/volumetypes_test.go

```diff
@@ -45,7 +45,7 @@ func TestVolumeTypesList(t *testing.T) {
 	}
 }
 
-func TestVolumeTypesCreate(t *testing.T) {
+func TestVolumeTypesCreateDestory(t *testing.T) {
```

Comment:
just a nit, but Destroy is misspelled, but to be consistent I would just change it to TestVolumeTypesCRUD.